### DISCUSSION TITLE
Another story. Really good. Please read

### DISCUSSION THREAD

**Discussion by frigus02** [frigus02/wfps2023-starter#1] [General]
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

**Comment by hybrist** [comment]
This reminds me of the [first story](https://github.com/frigus02/wpfs2023-starter/discussions/2)!

Or just plain https://github.com/frigus02/wpfs2023-starter/discussions/2.
